clusters/build-clusters/01_cluster/_machine_config/m5d_machineconfig.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: m5d
```

Comment:
A new one like this, or modify an existing one?

Reply:
What the effects to the current nodes?

Reply:
It should be fine to make a new machineset like this.  You can then perform a rollout by scaling down the old machineset and scaling up this one.